client/src/services/userService.js

```diff
@@ -22,37 +22,23 @@ class UserService {
     });
   }
 
-  // ToDo: Replace randomized name generator
   static whoami() {
-    const arr = ['admin', 'John Doe', 'Jane Doe'];
-
-    return this.authorizeUser().then(
-      response =>
-        new Promise(resolve => {
-          resolve({
-            userName: arr[Math.floor(Math.random() * arr.length)]
-          });
-        })
-    );
+    return axios({
+      method: 'get',
+      url: `${process.env.REACT_APP_USER_SERVICE}current/`,
```

Comment:
Optional: Didn't catch these before, and we can iterate on it... but the full url, probably needs to be placed in the ```dotenv``` files